pkg/content/file.go

```diff
@@ -440,3 +519,32 @@ func (w *fileWriter) Truncate(size int64) error {
 	}
 	return w.file.Truncate(0)
 }
+
+// pack given a bunch of descriptors, create a manifest that references all of them
+func pack(config ocispec.Descriptor, descriptors []ocispec.Descriptor) ([]byte, ocispec.Descriptor, error) {
+	if descriptors == nil {
+		descriptors = []ocispec.Descriptor{} // make it an empty array to prevent potential server-side bugs
+	}
+	// sort descriptors alphanumerically by sha hash so it always is consistent
+	sort.Slice(descriptors, func(i, j int) bool {
+		return descriptors[i].Digest < descriptors[j].Digest
+	})
```

Comment:
Why do we need to sort descriptors? We may need to keep the orignal order.

Related: https://github.com/oras-project/oras/issues/304

Reply:
Consistency. Although, in retrospect, I don't see why you might not want two distinct manifests for two distinct orderings. That is fair enough. If you want to open a PR to remove the sorting, I _think_ that would work.